artifactory/utils/transfersettings.go

```diff
@@ -0,0 +1,69 @@
+package utils
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+const transferSettingsFile = "transfer.conf"
+
+type TransferSettings struct {
+	ThreadsNumber int `json:"threadsNumber,omitempty"`
+}
+
+func ReadTransferSettings() (*TransferSettings, error) {
```

Comment:
ReadTransferSettings --> LoadTransferSettings
("Load" is the opposite of "Save" - so "LoadTransferSettings" is the opposite of the existing "SaveTransferSettings").